general-knowledge/data-collection/README.md

```diff
@@ -0,0 +1,20 @@
+# Data Collection
```

Comment:
Maybe some of the concepts/structure could be borrowed from my presentation here?
http://phontron.com/class/anlp2021/schedule/experimental-design.html